plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -209,7 +209,7 @@ def _configure_trials(
             )
             params = dict(trial.params)
             params.update(fixed_params)
-            overrides.append(tuple(f"{name}={val}" for name, val in params.items()))
+            overrides.append(tuple(f"++{name}={val}" for name, val in params.items()))
```

Comment:
This change has caused CI to fail. I can reproduce the issue locally.
The problem is that we might have e.g. `name == "~z", as was the case with the failing test. In that case the f-string `f"++{name}"` would be equal to `"++~z"`, which is no bueno.

I am not convinced that adding `++` is the right move here.

Reply:
I changed it back to remove '++', however I also extended the test to cover a case that I feel that must be supported (you can't define all parameters in advance in the DictConfig). Note that in the committed change 'w' cannot be overridden, because it's not known.

Reply:
Any ideas on how to resolve the currently failing test?

Reply:
I will take a look...

Reply:
@Jasha10 I've adapted the code to only add a '+' if the custom search space introduced a new parameter. Instead of by default add '++' to all parameters